Testscripts/Linux/verify_vmbus_interrupts.sh

```diff
@@ -45,9 +49,10 @@ function verify_vmbus_interrupts() {
     #
     # Verify if VMBUS interrupts are processed by all CPUs
     #
+    cat /proc/interrupts > interrupts
     while read line
     do
-        if [[ ($line = *hyperv* ) || ( $line = *Hypervisor* ) ]]; then
+        if [[ ($line = *Hyper-V* ) || ( $line = *Hypervisor* ) ]]; then
```

Comment:
The interrupt name has been changed from "hyperv" to "Hyper-V". Right?

Reply:
Yes